schemas/EGA.assay.json

```diff
@@ -0,0 +1,327 @@
+                  "required": [ "array_sample_labels" ],
+                  "properties": {
+                    "array_sample_labels": {
+                      "minItems": 3
+                    }
+                  }
+                }
+              },
+              {
+                "title": "4 labels per array check",
+                "description": "If four labels were used per array, the sample-label specifications will be expected and at least 4 items (one for each label)",
+                "if": {
+                  "required": [ "n_labels_per_array" ],
+                  "properties": {
+                    "prueba": { "const": 4 }
+                  }
+                },
+                "then": { 
+                  "required": [ "array_sample_labels" ],
+                  "properties": {
+                    "array_sample_labels": {
+                      "minItems": 4
+                    }
+                  }
+                }
+              }              
+            ]
+          },
+          "sequencing_assay_specifications": {
+            "type": "object",
+            "title": "Specifications of a sequencing assay",
+            "description": "Node containing the set of fields specific to an assay of type 'sequencing' (i.e. a sequencer was used to obtain the raw data).",
+            "additionalProperties": false,
+            "properties": {
+              "reference_alignment_details": {
+                "type": "array",
+                "title": "Reference assembly and sequence details",
+                "description": "Node containing details of the reference sequence used in the alignment.",
+                "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/reference_alignment_details"
+              }
+            }
+          }
+        },
+        "allOf": [
+          {
+            "title": "If the assay is of type array its specifications will be expected",
+            "if": {
+              "required": ["assay_type"],
+              "properties": {
+                "assay_type": {
+                      "enum": ["array"]
+                }
+              }
+            },
+            "then": {
+              "required": ["array_assay_specifications"]
+            }
+          }
+        ]
+      },
+
+      "assay_relationships": {
+        "type": "array",
+        "title": "Assay relationships",
+        "description": "Comprises metadata (e.g. Source or Target) of a directional association between two entities, one of which shall be the current assay.",
+        "minItems": 1,
+        "additionalProperties": false,
+        "uniqueItems": true,
+        "items": { 
+          "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/relationship_object" 
+        }
+      },
+
+      "assay_files": {
+        "type": "array",
+        "title": "Sequencing files from an assay",
```

Comment:
```suggestion
        "title": "Data files produced from an assay",
```

Reply:
Generalised to "Data files" given the files might come from an array assay, as well.

Reply:
Done